weblate/addons/base.py

```diff
@@ -148,20 +148,28 @@ class UpdateBaseAddon(BaseAddon):
     It hooks to post update and commits all changed translations.
     """
     events = (EVENT_POST_UPDATE, )
-    message = '''Update translation files
+    message = '''Update translation files{component}
 
-Updated by {name} hook in Weblate.'''
+Updated by "{name}" hook in Weblate.'''
 
     def update_translations(self, component, previous_head):
         raise NotImplementedError()
 
     def commit_and_push(self, component):
+        component_description = ''
+        if component.is_repo_link:
+            if component.linked_component.project != component.project:
+                component_description += ' ' + component.project.name + ' /'
+            component_description += ' ' + component.name
         repository = component.repository
         with repository.lock:
             if repository.needs_commit():
                 files = [t.filename for t in component.translation_set.all()]
                 repository.commit(
-                    self.message.format(name=self.verbose),
+                    self.message.format(
+                        name=self.verbose,
+                        component=component_description
+                    ),
```

Comment:
This would then become `weblate.utils.render.render_template(self.message, component=component, hook_name=name)`...